### PR TITLE
including service body counts

### PR DESCRIPTION
**Pull request by radius314**
This is a first pass at including service body counts.  I would like to change the display to be a tree view, but this will require a little more work.  However I feel this accomplishes the minimal goal of display the result from tomato.

### CODE REVIEW

**Comment by radius314** [comment]
image

**Comment by radius314** [comment]
image

File: index.html [hunk 1]
```diff
@@ -80,7 +81,35 @@ <h4><i>Contact admin@bmlt.app to get listed here.</i></h4>
...
+                <th><a href="#">Service Body</a></th>
+                <th><a href="#">Type</a></th>
+                <th><a href="#">Groups</a></th>
+                <th><a href="#">Meetings</a></th>
+            </tr>
+        </thead>
+        <tbody id="tallyBody">
+            {{#each this}}
+                <tr>
+                    <td>{{this.name}}</td>
```
Comment (pjaudiomv) [inline index.html]: I would add css ids here, then this could be aligned left and the rest right or whatever

**Comment by radius314** [comment]
left aligned now

**Comment by radius314** [comment]
Unknown

File: index.html (hunk 1 above)
Comment (ChrisMarshallNY) [reply index.html]: My thoughts exactly. If you can add a CSS ID that associates an ASC with its RSC (or Zone), then you can use CSS to establish a visual hierarchy.

Comment (ChrisMarshallNY) [reply index.html]: That also makes it SEP (Somebody Else's Problem).

**Comment by radius314** [comment]
The ID was added.  And this is an MVP, in the description of the pull request I mention the intention use a treeview.  CSS IDs will not make that happen automatically.

**Pull request merged**